packages/flutter/ios/sentry_flutter/Sources/sentry_flutter/SentryFlutterPlugin.swift

```
@@ -569,6 +450,98 @@ public class SentryFlutterPlugin: NSObject, FlutterPlugin {
+      let runtimeInitTimeMs = appStartMeasurement.runtimeInitTimestamp.timeIntervalSince1970.toMilliseconds()
+      let moduleInitializationTimeMs =
+          appStartMeasurement.moduleInitializationTimestamp.timeIntervalSince1970.toMilliseconds()
+      let sdkStartTimeMs = appStartMeasurement.sdkStartTimestamp.timeIntervalSince1970.toMilliseconds()
+
+      if !appStartMeasurement.isPreWarmed {
+          let preRuntimeInitDescription = "Pre Runtime Init"
+          let preRuntimeInitSpan: [String: Any] = [
+              "startTimestampMsSinceEpoch": NSNumber(value: appStartTimeMs),
+              "stopTimestampMsSinceEpoch": NSNumber(value: runtimeInitTimeMs)
+          ]
+          nativeSpanTimes[preRuntimeInitDescription] = preRuntimeInitSpan
+
+          let moduleInitializationDescription = "Runtime init to Pre Main initializers"
+          let moduleInitializationSpan: [String: Any] = [
+              "startTimestampMsSinceEpoch": NSNumber(value: runtimeInitTimeMs),
+              "stopTimestampMsSinceEpoch": NSNumber(value: moduleInitializationTimeMs)
+          ]
+          nativeSpanTimes[moduleInitializationDescription] = moduleInitializationSpan
+      }
+
+      let uiKitInitDescription = "UIKit init"
+      let uiKitInitSpan: [String: Any] = [
+          "startTimestampMsSinceEpoch": NSNumber(value: moduleInitializationTimeMs),
+          "stopTimestampMsSinceEpoch": NSNumber(value: sdkStartTimeMs)
+      ]
+      nativeSpanTimes[uiKitInitDescription] = uiKitInitSpan
+
+      let appStartTime = appStartMeasurement.appStartTimestamp.timeIntervalSince1970 * 1000
+      let isColdStart = appStartMeasurement.type == .cold
+
+      let item: [String: Any] = [
+          "pluginRegistrationTime": pluginRegistrationTime,
+          "appStartTime": appStartTime,
+          "isColdStart": isColdStart,
+          "nativeSpanTimes": nativeSpanTimes
+      ]
+
+      if let data = try? JSONSerialization.data(withJSONObject: item, options: []) {
```

We could use do/catch and log the error. Now we fail silently.

---

done